README.md

```diff
@@ -6,5 +6,86 @@ This is an Android SDK for the [KUKSA Vehicle Abstraction Layer](https://github.
 [![License](https://img.shields.io/badge/License-Apache%202.0-green.svg)](https://opensource.org/licenses/Apache-2.0)
 [![Gitter](https://img.shields.io/gitter/room/kuksa-val/community)](https://gitter.im/kuksa-val/community)
 
+## Overview
 
-It allows you to access VSS data from KUKSA databroker using an Android System.
+The Kuksa Android SDK allows you to interact with [VSS data](https://github.com/COVESA/vehicle_signal_specification) 
+from the [KUKSA Databroker](https://github.com/eclipse/kuksa.val/tree/master/kuksa_databroker)
```

Comment:
I am a little bit disagreeing here because everyone would just go instantly to the sub link from the main page. Feels more like the right entry point to me.

Reply:
Removed the last sentence and linked to GitHub packet